app/javascript/gobierto_data/webapp/components/sets/Nav.vue

```diff
@@ -171,15 +153,107 @@ export default {
 
     this.$root.$on('changeNavTab', this.changeTab)
     this.$root.$on('activeTabIndex', this.changeTab)
+    this.$root.$on('reloadQueries', this.getQueries)
 
     this.slugName = this.$route.params.id
+    this.userId = getUserId()
+    this.token = getToken()
+
+    this.setValuesDataset()
   },
   methods: {
     changeTab() {
       this.activateTab(1)
     },
     activateTab(index) {
+      this.activeTab = index
       this.$emit("active-tab", index);
+    },
+    setValuesDataset(){
+      const url = `${baseUrl}/datasets/${this.slugName}/meta`
+      axios
+        .get(url)
+        .then(response => {
+
+         const rawData = response.data
+         const { data: {
+           id: datasetId,
+           attributes: {
+             name: titleDataset,
+             slug: slugDataset,
+             table_name: tableName,
+             columns: arrayColumns,
+             description: descriptionDataset,
+             data_summary: {
+               number_of_rows: numberRows
+             },
+             formats: arrayFormats
+           }
+         } } = rawData;
+
+          const resourcesData = response.included
+          this.datasetId = parseInt(datasetId)
+          this.titleDataset = titleDataset
+          this.slugDataset = slugDataset
+          this.tableName = tableName
+          this.arrayFormats = arrayFormats
+          this.arrayColumns = arrayColumns
+          this.numberRows = numberRows
+          this.descriptionDataset = descriptionDataset
+          const dateFromApi = rawData.data.attributes.data_updated_at === undefined ? '' : rawData.data.attributes.data_updated_at
```

Comment:
Why in the heck, after the most large object destructuring I've ever seen, do you use again the properties in that way `rawData.data.attributes.data_updated_at`?

The fun part of destructuring is that you can initialize the objects, to avoid errors